pkg/validation/validate_cluster.go

```diff
@@ -299,8 +299,11 @@ func (v *ValidationCluster) validateNodes(cloudGroups map[string]*cloudinstances
 		}
 
 		for _, member := range allMembers {
-			node := member.Node
+			if member.Status == cloudinstances.CloudInstanceStatusDetached {
+				continue
+			}
```

Comment:
@olemarkus @johngmyers Should the bastion and warm pool checks be handled in the same way?

Reply:
Bastions and warm instances will never join, so the node readiness and critical pod checks are never relevant for them.